TopicZero/.gitignore

```diff
@@ -0,0 +1 @@
+/bin/
```

Comment:
To avoid to push to the repo Eclipse configuration files you can add in .gitignore the following lines: .classpath, .project and .settings. It is just a comment to have in account in next topics